app/src/main/java/amazingme/app/UserContext.java

```diff
@@ -10,23 +11,22 @@
 public class UserContext {
 
     private Parent parent;
-    private List<Child> children; //not setting this as final because it could change in a session of using the app
-    private List<GameResult> gameResults;
+    private Map<String, Child> children; //not setting this as final because it could change in a session of using the app
```

Comment:
i think it had something to do with not wanting to concern myself with which child was where in a list. i'm trying to remember why I ended up doing that. 

it also gives us the flexibility to get their info by their name, if that ever becomes easier. i toyed with that idea for awhile but thought the backend would better be handled using their index. and, unfortunately, you're not allowed to use a map with a non-string key in firebase (if i remember the error correctly).

Reply:
okay. i was just wondering since you end up concatenating the index with a token anyway. so i didnt see any difference with just indexing a list (like the index is sorta the children id). we can keep this change i dont think it makes a huge difference